java/ci/build-in-docker.sh

```diff
@@ -78,7 +78,6 @@ BUILD_ARG="-Dmaven.repo.local=\"$WORKSPACE/.m2\"\
  -DPER_THREAD_DEFAULT_STREAM=$ENABLE_PTDS\
  -DCUDA_STATIC_RUNTIME=$ENABLE_CUDA_STATIC_RUNTIME\
  -DCUDF_JNI_LIBCUDF_STATIC=ON\
- -DRMM_LOGGING_LEVEL=$RMM_LOGGING_LEVEL\
```

Comment:
Do we need to make similar changes in [build.sh](https://github.com/rapidsai/cudf/blob/branch-22.06/build.sh#L151)?

Reply:
Nice catch!  This should now be fixed.